src/components/map/map.tsx

```diff
@@ -0,0 +1,94 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Listen,
+  Prop
+} from "@stencil/core";
+import { IComponent } from "../common/interfaces";
+import {
+  Marker,
+  map as LFMap,
+  tileLayer
+  // tslint:disable-next-line:no-submodule-imports
+} from "leaflet/dist/leaflet-src.esm";
+
+@Component({
+  shadow: false,
+  styleUrl: "map.scss",
+  tag: "gx-map"
+})
+export class Map implements IComponent {
+  map: any;
+  @Element() element;
+
+  /**
+   * The initial center of the map.
+   *
+   */
+  @Prop({ mutable: true })
+  center = "0, 0";
+
+  /**
+   * The max zoom scale aviable in the map.
+   *
+   */
+  @Prop({ mutable: true })
```

Comment:
This property is never modified, so it shouldn't really be mutable, right?

Reply:
Ready.